realworld-OOP-lab/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+	"presets": ["es2015"]
+}
```

Comment:
Always leave a new line at the end of your files

Reply:
noted

Reply:
noted.